### PR TITLE
update reports urls

### PR DESCRIPTION
**Pull request by nickpell**

### CODE REVIEW

File: corehq/apps/reports/urls.py [hunk 1]
```diff
@@ -56,84 +67,84 @@
...
 
     # export API
-    url(r"^export/$", 'export_data'),
+    url(r"^export/$", export_data, name='export_data'),
 
     # Download Exports
     # todo should eventually be moved to corehq.apps.export
     # Custom
-    url(r"^export/custom/(?P<export_id>[\w\-]+)/download/$", 'export_default_or_custom_data', name="export_custom_data"),
+    url(r"^export/custom/(?P<export_id>[\w\-]+)/download/$", export_default_or_custom_data, name="export_custom_data"),
```
Comment (dimagimon) [inline corehq/apps/reports/urls.py]: E501 line too long (119 > 115 characters)

File: corehq/apps/reports/urls.py [hunk 2]
```diff
@@ -56,84 +67,84 @@
...
     # Default
-    url(r"^export/default/download/$", "export_default_or_custom_data", name="export_default_data"),
+    url(r"^export/default/download/$", export_default_or_custom_data, name="export_default_data"),
     # Bulk
-    url(r"^export/bulk/download/$", "export_default_or_custom_data", name="export_bulk_download", kwargs=dict(bulk_export=True)),
+    url(r"^export/bulk/download/$", export_default_or_custom_data, name="export_bulk_download", kwargs=dict(bulk_export=True)),
```
Comment (dimagimon) [inline corehq/apps/reports/urls.py]: E501 line too long (127 > 115 characters)

File: corehq/apps/reports/urls.py [hunk 3]
```diff
@@ -56,84 +67,84 @@
...
     # saved
-    url(r"^export/saved/download/(?P<export_id>[\w\-]+)/$", "hq_download_saved_export", name="hq_download_saved_export"),
-    url(r"^export/saved/download/deid/(?P<export_id>[\w\-]+)/$", "hq_deid_download_saved_export", name="hq_deid_download_saved_export"),
-    url(r"^export/saved/update/$", "hq_update_saved_export", name="hq_update_saved_export"),
+    url(r"^export/saved/download/(?P<export_id>[\w\-]+)/$", hq_download_saved_export, name="hq_download_saved_export"),
+    url(r"^export/saved/download/deid/(?P<export_id>[\w\-]+)/$", hq_deid_download_saved_export, name="hq_deid_download_saved_export"),
```
Comment (dimagimon) [inline corehq/apps/reports/urls.py]: E501 line too long (134 > 115 characters)

**Comment by nickpell** [comment]
no issues noticed on staging

@benrudolph @emord Let's try this PR today.  I'll be around in the evening in case anything goes wrong.

**Comment by benrudolph** [comment]
you got it

**Pull request merged**